test/unit/inbound/save-schedule.test.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../../../app/data')

```

Comment:
I'm rethinking my original, "add unit tests" comment.

Because this is nearly 90% Sequelize db save downs, I think we can just make these integration tests.

This one already is becuz we aren't mocking the `db` config for e.g. so can just move this file under integration folder